codespell_lib/data/dictionary.txt

```diff
@@ -29060,6 +29094,8 @@ initates->initiates, imitates,
 initating->initiating, imitating,
 initation->initiation, imitation,
 initations->initiations, imitations,
+initative->initiative, imitative,
```

Comment:
Again, no typo of `indicative`? Perhaps even `limitative`?

Reply:
I couldn't find these typos.